environmental_data_modules/aurn_extractor.py

```diff
@@ -185,7 +185,7 @@ def define_years_to_download(self, subset_df, years):
         datetime_start = pd.to_datetime(subset_df['start_date'].values, format='%Y/%m/%d').year
         # Problem with the end date is it could be ongoing. In which case, convert that entry into a date
```

Comment:
Lines 183 above, use of 'your'. Do a search on 'Your' and (e.g.) 'our' and replace with objective description of variable names.

Reply:
I have reviewed the comments in the code - they were rather unhelpful! I've made them more relevant to what the code is doing (and more objective).